### ISSUE TITLE
ipsw extract --dmg not doing anything?

### ISSUE THREAD

**Issue by pwmoore** [blacktop/ipsw#229]
I want to use `ipsw` to extract the App and OS DMG files from downloaded IPSW.

I'm issuing the command:

```
$ ipsw extract -m <STRING> <IPSW>
```

Where `<STRING>` is what I want to extract and `<IPSW>` is the IPSW file. For `<STRING>` I've tried `app`, `sys`, as well as the filename of the DMG. I've also tried specifying an output directory and moving arguments around. 

No matter what I try, I get no output from `ipsw`, even with verbose mode enabled. The return code from `ipsw` is 0 as well.

It's entirely possible this is user error on my part, so if I'm doing something wrong, just slap me upside the head :)

**Comment by blacktop** [comment]
Phil! how's it going?!

Many thanks for pointing this out to me ❤️

**Comment by blacktop** [comment]
should be fixed in latest release. 👍 (one of these days I'll write a comprehensive regression test suite, but until then I'll have to continue relying on the kindness of strangers 😉)

**Comment by pwmoore** [comment]
Just got around to testing it, it worked a charm! Many thanks as always! 

On an unrelated note, I swear, I can't tell you how often myself or a co-worker needs to do something iOS related, and before I try to do it myself, I think "Wait, I bet`ipsw` already has that feature!"

I was looking at the RSR that was released last week and wanted to figure out how to recover the DSC without necessarily having to install it... I checked `ipsw` and sure enough, it does exactly what I need it to! I'm full on evangelical about this tool, it makes my life so much better.
